head/core/tasks_proto.cpp

```diff
@@ -70,6 +70,10 @@ static auto i2c3_poll_client =
     i2c::poller::Poller<freertos_message_queue::FreeRTOSMessageQueue>{};
 static auto eeprom_task_builder =
```

Comment:
Let's disable this on non-DVT head boards for mfg error reasons

Reply:
We don't have the issue on the A rev boards since they use a different eeprom chip than the Rev B boards